CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## TBD
+
+### Enhancements
+
+* Added ApiKey to session and it can be changed in an `OnSessionCallback`.
+  [#1855](https://github.com/bugsnag/bugsnag-android/pull/1855)
+* 
```

Comment:
```suggestion
* Added `Session.apiKey` so that it can be changed in an `OnSessionCallback`.
  [#1855](https://github.com/bugsnag/bugsnag-android/pull/1855)

```